tests/Doctrine/Tests/DBAL/Schema/ColumnTest.php

```diff
@@ -62,7 +62,8 @@ public function testToArray()
      */
     public function testSettingUnknownOptionIsStillSupported() : void
     {
-        new Column('foo', $this->createMock(Type::class), ['unknown_option' => 'bar']);
+        $col1 = new Column('foo', $this->createMock(Type::class), ['unknown_option' => 'bar']);
+        self::assertInstanceOf(Column::class, $col1);
```

Comment:
This assertion is redundant. The `new` operator cannot produce any else than an instance of the requested class.

Reply:
based on the test function, i was guessing that this was testing that the object constructor would not throw an exception if you pass 'unknown_option'. The pass is that the $col1 is correctly a Column. Otherwise, my local php unit was giving an alert that the test has no assertion, so no test was actually being performed.

Reply:
It has something to do with the Symfony bridge for PHPUnit (https://github.com/doctrine/dbal/pull/2846). It should take care of this case. Please omit this change as long as it's not part of the fix you're working on.

Reply:
reverted.

Reply:
reverted.